### PR TITLE
Fixes to be able to build with latest XCode

### PR DESCRIPTION
**Pull request by mathbunnyru**
Deleted Sparkle (your app doesn't update anyway). Made some fixes to make it work with latest Xcode.

### CODE REVIEW

**Comment by mathbunnyru** [comment]
I don't know ObjectiveC very well. There are some deprecated functions in your code. If you could look at them, it would be awesome.

File: LoginItemsAE.c [hunk 1]
```diff
@@ -94,7 +94,7 @@ static OSStatus LaunchSystemEvents(ProcessSerialNumber *psnPtr)
     // Launch it!
     
     if (err == noErr) {
-        if ( LSOpenApplication != NULL ) {
+        if ( &LSOpenApplication != NULL ) {
```
Comment (taoeffect) [inline LoginItemsAE.c]: I don't have time to look into it right now, but this seems possibly incorrect? Then again, I didn't write this code, so maybe it's fine.

**Comment by mathbunnyru** [comment]
@taoeffect I made all changes possible (actually I was studying Objective-C for several minutes). Hope you'll merge these changes. Big changes - the project now compiles with latest Xcode, there's no "Check for updates option" (I'm more than sure, that this simple project doesn't need it). I used latest CTGradient and LoginItems I've found online. And I've made some warnings disappear. There are a few of them remaining. If you could look at them yourself, that would be awesome. Thank you.

**Comment by taoeffect** [comment]
These changes look OK to me, however, re this:

> And I've made some warnings disappear.

You also removed `-Wall`. That's not a legitimate way of making warnings disappear. :P

If you can add that back I'll merge it.

**Comment by taoeffect** [comment]
Congrats, great job! 👍

**Comment by mathbunnyru** [comment]
@taoeffect thanks:) I already see the benefit of the job done - now the app opens and quits more smoothly on latest OS X. I recommend you to update version number and to make a new release (and to update brew cask for your app and a lot of guys will receive the update).

**Comment by taoeffect** [comment]
@mathbunnyru Those are good ideas, but I haven't worked on DelayedLauncher in a long time, and there are many other tasks demanding my attention. If there were more significant changes submitted I might be able to justify the time spent on creating a cask for this, but honestly IMO this was one step forward and one step backward because of the removal of Sparkle. Most OS X users who use DelayedLauncher rely on Sparkle or MAS to update their apps—not homebrew, which is for developers.

I OK'd your PR simply because you're the first to give a stab at improving DelayedLauncher, and I respect that. If you want to work on it some more, and maybe add the latest version of Sparkle, that might be worth me doing a release for it.

In the meantime, you are free to add your own build to Homebrew/Cask if you'd like, just please make sure to include proper credit and a link back to this repo.

**Comment by mathbunnyru** [comment]
Ok, I completely understand. Btw, cask already exists: https://github.com/caskroom/homebrew-cask/blob/master/Casks/delayedlauncher.rb

**Pull request merged**